pagopa/io-app
# chore(IT Wallet): [SIW-1793] Update non-matching identity screen

## Short description
This PR updates the screen shown when the user who is authenticating is not the same that activated the wallet.

## List of changes proposed in this pull request
- Updated `ITW_IDENTITY_NOT_MATCHING_SCREEN`

## How to test
Authenticate with a different identity than the one used to activate the wallet. You should see the following screen:

<img src="https://github.com/user-attachments/assets/84b84dd5-7c80-4ef6-97e6-75c4f387fe9f" width="240" />
<img src="https://github.com/user-attachments/assets/d1073c2b-584a-46f4-94eb-08b93e3cc1f1" width="240" />
